apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/AssemblyInfo.cs

```diff
@@ -14,10 +14,30 @@
 
 using System.Runtime.CompilerServices;
 
-[assembly: InternalsVisibleTo("Google.Cloud.Spanner.Data.IntegrationTests, PublicKey=" 
+[assembly: InternalsVisibleTo("Google.Cloud.Spanner.Data.IntegrationTests, PublicKey="
                               + "0024000004800000940000000602000000240000525341310004000001000100afab79952ee222"
                               + "15f12b4e09337e65509c943fbc22d7006bc371d581d0f0ebf0da5d8039aab2607fb68a138a5d80"
                               + "a71bc02b7ebf586dbe1f2493c0ab20423ababfd15ce74d2264a6b37745f3658f016abaad662182"
                               + "aaef634a60f1346fcc45343acab5b6781535a3134818e13fac895a6c106c0480e34bbb06cb123e"
                               + "5583d8d2"
 )]
+
+[assembly: InternalsVisibleTo("Google.Cloud.Spanner.Data.Tests, PublicKey="
+                              + "0024000004800000940000000602000000240000525341310004000001000100afab79952ee222"
+                              + "15f12b4e09337e65509c943fbc22d7006bc371d581d0f0ebf0da5d8039aab2607fb68a138a5d80"
+                              + "a71bc02b7ebf586dbe1f2493c0ab20423ababfd15ce74d2264a6b37745f3658f016abaad662182"
+                              + "aaef634a60f1346fcc45343acab5b6781535a3134818e13fac895a6c106c0480e34bbb06cb123e"
+                              + "5583d8d2"
+)]
+
+[assembly: InternalsVisibleTo("Castle.Core, PublicKey=002400000480000094000000060200000024000052534131000400000"
```

Comment:
its been removed and was only needed in the pool tests for the spannerclientfactory, where I made my own mock class.